budgeteer-web-interface/src/main/java/org/wickedsource/budgeteer/service/project/ProjectService.java

```diff
@@ -63,9 +63,14 @@
      * @param projectName name of the project.
      * @return the base data of the newly create project.
      */
-    public ProjectBaseData createProject(String projectName, long initialUserId) {
+    public ProjectBaseData createProject(String projectName, long initialUserId) throws Exception {
         UserEntity user = userRepository.findOne(initialUserId);
         ProjectEntity project = new ProjectEntity();
+        for(ProjectEntity e : projectRepository.findAll()){
+            if(e.getName().equals(projectName)){
+                throw new Exception("A");
```

Comment:
The message for this exception could be a little more explaining, don't you think? 😄

Reply:
Changed in new PR